overlord/ifacestate/ifacestate_test.go

```diff
@@ -2523,6 +2795,72 @@ slots:
 	c.Check(ifacestate.CheckInterfaces(s.state, snapInfo), IsNil)
 }
 
+func (s *interfaceManagerSuite) TestCheckInterfacesDeviceScopeRightStore(c *C) {
```

Comment:
Same here wrt adding nice comments. Also, I feel like these tests that are looking at installation constraints should be names with 'Installation' in the name. Not a blocker.